qdrant/qdrant_grpc.pb_test.go

```diff
@@ -21,7 +21,7 @@ func TestNewQdrantClient(t *testing.T) {
 		distance              = pb.Distance_Dot
 	)
 
-	c, err := qdrant.RunContainer(context.Background(), testcontainers.WithImage("qdrant/qdrant:v1.8.1"))
+	c, err := qdrant.RunContainer(context.Background(), testcontainers.WithImage("qdrant/qdrant:dev"))
```

Comment:
We probably need to put this back to v1.9 on release, don't we?

Just putting a marker here!

Reply:
dear sir, please read the PR description :smile_cat: